packages/governance/test/swingsetTests/contractGovernor/bootstrap.js

```diff
@@ -390,7 +390,7 @@ const makeBootstrap = (argv, cb, vatPowers) => async (vats, devices) => {
 
       await E(timer).tick();
       await E(timer).tick();
-      E.when(
+      await E.when(
```

Comment:
```suggestion
      void E.when(
```

The test doesn't want to wait here.  It wants these logs requests to fire later.

Reply:
thanks, I was wondering why they were failing ;-)